src/main/java/uk/co/onsdigital/discovery/metadata/api/dto/Edition.java

```diff
@@ -0,0 +1,34 @@
+package uk.co.onsdigital.discovery.metadata.api.dto;
+
+import java.util.List;
+
+public class Edition {
```

Comment:
Needs a toString method, as it's included in the toString of DimensionalDataSetResult